app/repos.rb

```diff
@@ -19,20 +19,4 @@ def self.active
   def self.active_apps
     Repos.active.select(&:is_app?)
   end
-
-  def self.grouped_by_team
-    Repos.active.reject(&:private_repo?).group_by(&:team)
-  end
-
-  def self.teams
-    Repos.grouped_by_team.keys.reject { |team| team == UNKNOWN }
-  end
-
-  def self.for_team(team)
-    Repos.grouped_by_team.fetch(team, []).map(&:app_name)
-  end
-
-  def self.no_known_owner
-    Repos.for_team(UNKNOWN)
-  end
```

Comment:
Note that I spent some time trying to re-create ActiveRecord style 'scope' queries for these, before concluding that actually many of these methods can just be removed.

Here is the commit to the 'scope' spike work, for future reference: https://github.com/alphagov/govuk-developer-docs/commit/1a19c5bc1f60a698b383ebcc6736fc3a8abb6503

Reply:
Note to self: split this PR into maybe 3:

1 for the general refactoring
1 for the Apps/Repos pages changes
1 for the Repo overview page changes